src/compilerv2/compilerv2.c

```
@@ -0,0 +1,197 @@
+
+				ctx.variableCount++;
+				break;	
+			case AST_FUNCTION_DEF:
+				struct Function* func = malloc(sizeof(struct Function));
+
+				func->name = node->left->left->value;
+
+				while(node->left->right->next != NULL) {
+					node->left->right = node->left->right->next;
+
+					int c = func->variableCount;
```

cppcheck (2.10-2)

[error] Uninitialized struct member

(uninitStructMember)